pro/src/routes/OfferIndividualCreation/OfferIndividualCreation.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Switch, Route, Redirect } from 'react-router-dom'
+
+import Breadcrumb, { BreadcrumbStyle } from 'new_components/Breadcrumb'
+import { OfferFormLayout } from 'new_components/OfferFormLayout'
+
+import { useOfferFormSteps } from 'core/Offers'
+import { fakeOffer } from 'screens/OfferIndividual/constants'
+import { OfferIndividualCreationInformations as InformationsRoute } from 'routes/OfferIndividualCreationInformations'
+import { OfferIndividualCreationStocks as StockRoute } from 'routes/OfferIndividualCreationStocks'
+import { OfferIndividualCreationSummary as ConfirmationRoute } from 'routes/OfferIndividualCreationSummary'
+
+const OfferIndividualCreation = (): JSX.Element => {
+  const { currentStep, stepList } = useOfferFormSteps(fakeOffer)
+
+  return (
+    <OfferFormLayout>
+      <OfferFormLayout.TitleBlock>
+        <h1>Créer une offe</h1>
+      </OfferFormLayout.TitleBlock>
+
+      <OfferFormLayout.Stepper>
+        <Breadcrumb
+          activeStep={currentStep.id}
+          steps={stepList}
+          styleType={BreadcrumbStyle.TAB}
+        />
+      </OfferFormLayout.Stepper>
+
+      <OfferFormLayout.Content>
+        <Switch>
+          <Route
+            exact
+            path={[
+              '/offre/v3/creation/individuelle/informations',
+              '/offre/:offerId/v3/creation/individuelle/informations',
+            ]}
+          >
```

Comment:
On avait commencé à en discuter sur la première PR avant que tu enlève temporairement ce niveau :
Je trouve ça risqué d'avoir deux systèmes différents pour lire dans l'url et en déduire l'étape du formulaire.

Que va-t-il se passer en cas de désynchro ? (bug d'un côté ou de l'autre, ou config pas complètement alignée)
ça ne va pas donner un simple bug mais ça pourrait conduire à des comportements vraiment bizarres et difficiles à analyser.

Je pense qu'il faut qu'on respecte le principe du Single Source of Truth,
surtout que là c'est facile c'est au sein du même composant !
Tu pourrais par exemple faire un switch/case javascript au lieu de celui en React-Router. (De toute façon derrière c'est à peu prêt ce que fait React-Router : un matchPath et un switch/case js, sauf qu'il le fait avec ses propres règles de calcul plutôt que les notres d'où le risque)

Reply:
note: remplacer router.switch par switch ou if/else